src/__tests__/helpers.ts

```diff
@@ -1,18 +1,31 @@
+import { MutatorFn } from 'src/serializers';
 import {
   Pipeline,
   Serializer,
   serializers as predefinedSerializers,
+  Step,
+  CommandStep,
 } from '../';
 import { resetUuidCounter } from './setup';
+import seedrandom from 'seedrandom';
 
 type SerializerType =
   | 'json'
   | 'json_depends_on'
   | 'yaml'
   | 'yaml_depends_on'
   | 'dot'
-  | 'structure';
+  | 'structure'
+  | 'yaml_mutate';
 
+const mutate: MutatorFn = async (entity: Step) => {
+  if (entity instanceof CommandStep) {
+    const random = seedrandom(
+      JSON.stringify(entity.command.map((command) => command.serialize())),
+    );
+    entity.withKey(`key-${random.int32()}`);
```

Comment:
I noticed a couple of things:

some keys are:
```
  - label: Test Editor
    key: key--820674632
```
as `int32` returns a signed result. Probably okay, just not very obvious.

And then I noticed the autogenerated keys via `_<int>`.
I would like to make it more obvious when we are looking at mutated keys:

```suggestion
    entity.withKey(`mutated_${random.int32()}`);
```

WDYT?

Reply:
SGTM